e2e/deployers/appset.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/ramendr/ramen/e2e/config"
 	"github.com/ramendr/ramen/e2e/types"
 	"github.com/ramendr/ramen/e2e/util"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
We use also the standard errors package. Please use k8serrors when importing this package even if we don't use the stdlib errors package in a file. We want to use the same name for the same library in the entire project. This should be correct now in ramen code itself.